src/components/movie-tile/movie-tile.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './movie-tile.scss';
+import PropTypes from 'prop-types';
+import ContextMenu from '../common/context-menu/context-menu';
+
+const MovieTile = (props) => {
+  const { imgUrl, imgAlt, title, year, genreList } = props.movie;
```

Comment:
I don't see a reason for destructuring only a part of props object (you use props.{property} below in code). And also you can destructure values direcly in the function params